### PR TITLE
Feat/launch/head template

### PR DESCRIPTION
**Pull request by bennypowers**
## What I did

1. fix the mobile menu button - uses `color: currentColor` for themability
2. add noscript.css to assets, and add a fix in launch preset for an 11ty bug
3. remove `_assets/head.html` and add `_includes/partials/head.njk`
4. lint layout.css with stylelint

### CODE REVIEW

File: packages/launch/theme/rocketLaunch.js [hunk 1]
```diff
@@ -47,5 +47,23 @@ export function rocketLaunch() {
...
+       * - `(\s+)?`: any amount of whitespace or none
+       * - `(&.*;)`: any html entity e.g. &#25;
+       * - `link`: the literal string 'link'
+       *
+       * In other words, the URL-encoded double-encoded string `<link`
+       */
+      const ENCODED_LINK_TAG_RE = /((\s+)?(&.*;)link)/;
+
+      eleventyConfig.addTransform('fix-noscript-style', content =>
+        content.replace(ENCODED_LINK_TAG_RE, '<link'),
```
Comment (daKmoR) [inline packages/launch/theme/rocketLaunch.js:65]: seems to work fine for me without this? 🤔

Comment (bennypowers) [reply packages/launch/theme/rocketLaunch.js:65]: yeah LGTM without it, too. You can remove

Comment (bennypowers) [reply packages/launch/theme/rocketLaunch.js:65]: this is ready to merge

**Comment by daKmoR** [comment]
sweet thx 👍

**Pull request merged**